src/ctia/entity/event.clj

```diff
@@ -143,22 +143,28 @@
    (event-history-routes services)
    (services->entity-crud-routes
     services
-    {:tags ["Event"]
-     :entity :event
-     :entity-schema Event
-     :get-schema PartialEvent
-     :get-params EventGetParams
-     :list-schema PartialEventList
-     :search-schema PartialEventList
-     :search-q-params EventSearchParams
-     :get-capabilities :read-event
-     :can-update? false
-     :can-patch? false
-     :can-post? false
+    {:tags                    ["Event"]
+     :entity                  :event
+     :entity-schema           Event
+     :get-schema              PartialEvent
+     :get-params              EventGetParams
+     :list-schema             PartialEventList
+     :search-schema           PartialEventList
+     :search-q-params         EventSearchParams
+     :get-capabilities        :read-event
+     :can-update?             false
+     :can-patch?              false
+     :can-post?               false
      :can-get-by-external-id? false
-     :search-capabilities :search-event
-     :delete-capabilities #{:delete-event :developer}
-     :date-field :timestamp})))
+     :search-capabilities     :search-event
+     :delete-capabilities     #{:delete-event :developer}
+     :date-field              :timestamp
+     :searchable-fields       #{:id
+                                :fields.change.after
+                                :event_type
+                                :fields.action
+                                :fields.change.before
+                                :fields.field}})))
```

Comment:
only some fields are indexed in events: https://github.com/threatgrid/ctia/blob/43ff86881b099bfd5a86f6a24a7ec0e05e980c04/src/ctia/entity/event.clj#L36
```suggestion
     :searchable-fields       #{:id
                                :source
                                :entity.id
                                :event_type}})))
```